README.md

```diff
@@ -0,0 +1,30 @@
+## Purpose
+
+The purpose is to achieve uniformity to create taxonomies from different information sources 
+and extract data that is relevant to our format and at the end create from them JSON in our format.
+
+## SNOMED CT
+SNOMED CT (Systematized Nomenclature of Medicine - Clinical Terms) is a standardized, multilingual vocabulary of clinical terminology.
```

Comment:
An alternative with some extra information (to replace lines 6-14):
```
## SNOMED CT

SNOMED CT (Systematized Nomenclature of Medicine - Clinical Terms) is a standardized, multilingual vocabulary of clinical terminology.
In SNOMED there are Concepts and Relations.

### Concepts

Each concept may have multiple names, including "fully specified names" (names that uniquely identify the concept) and "synonyms" (other names, sometimes more in use). We each of these groups has a single preferred name.
SNOMED treats the **preferred synonym** as the canonical name and so do we. The rest are used as aliases resulting in 3 groups:
- **synonyms**: the names of the "synonyms" group (except the "preferred synonym")
- **preferred fully specified name**
- **fully specified names**:  the names of the "fully specified names" group (except the "preferred synonym")

We filter out concepts that are not marked as active. 

### Relations

There are many kinds of relations between concepts, we chose to adopt the "is a" relation which SNOMED itself uses to define the taxonomy hierarchy.

In the "is a" relation hierarchy, concepts may have more than one father, so we performed the following heuristic priority to linearize the graph (force each concept to have one father at most):

\```
Choose the father of a concept according to the following priority:
(1) a descendant of a disease 
(2) a descendant of clinical findings
(3) the number of his children
\```

```
Notice to remove the slash before the backticks (**\\```**).
Here is how it looks at my pycharm:
![image](https://user-images.githubusercontent.com/5661975/184123763-d3712ec5-abc4-4a6f-8b19-484010ea04cd.png)